ui.tests/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>com.adobe.aem</groupId>
         <artifactId>core-forms-components-parent</artifactId>
-        <version>2.0.100-SNAPSHOT</version>
+        <version>2.0.19-SNAPSHOT</version>
```

Comment:
There is no point changing this version. Please remove the other changes and keep this PR only for changes mentioned in the PR description.